app/lib/shared/mock_scores.json

```diff
@@ -0,0 +1,1182 @@
+ "libffi6_extension": 0.0008538712471736237,
+ "libffi_binaries": 0.0008538712471736237,
+ "parser_error": 0.0008538712471736237,
+ "macro_processor": 0.0008538712471736237,
+ "map_parser": 0.0008538712471736237,
+ "binary_marshalling": 0.0008538712471736237,
+ "binary_interop": 0.0008538712471736237,
+ "game_loop": 0.0006830974429182884,
+ "reddit": 0.0006830974429182884,
+ "stevenroose": 0.0006830974429182884,
+ "custom_element": 0.0006830974429182884,
+ "stats": 0.0006830974429182884,
+ "clone": 0.0006830974429182884,
+ "protractor": 0.0006830974429182884,
+ "dliver": 0.0006830974429182884,
+ "polymer_intl": 0.0006830974429182884,
+ "json_compressor": 0.0006830974429182884,
+ "smooth_scroll": 0.0006830974429182884,
+ "scroll": 0.0006830974429182884,
+ "scissors": 0.0006830974429182884,
+ "angel_serialize_builder": 0.0006830974429182884,
+ "dart_constructs": 0.0006830974429182884,
+ "tuneup": 0.0006830974429182884,
+ "md_proc": 0.0006830974429182884,
+ "millisecond": 0.0006830974429182884,
+ "mock_web_server": 0.0006830974429182884,
+ "template_cache_transformer": 0.0006830974429182884,
+ "polymer_ui_elements": 0.0005123233417893187,
+ "sparse_list": 0.0005123233417893187,
+ "mojo_services": 0.00034154901810408287,
+ "braxter_common": 0.00034154901810408287,
+ "html_import": 0.00034154901810408287,
+ "levenshtein": 0.00034154901810408287,
+ "clean_data": 0.00034154901810408287,
+ "jsonify": 0.00034154901810408287,
+ "twit": 0.00034154901810408287,
+ "fancy_syntax": 0.00034154901810408287,
+ "character_parser": 0.00034154901810408287,
+ "rethinkdb_driver2": 0.00034154901810408287,
+ "wired": 0.00034154901810408287,
+ "mirrorme": 0.00034154901810408287,
+ "mojo_sdk": 0.00034154901810408287,
+ "longdouble": 0.00017077454611292685,
+ "animate": 0.00017077454611292685,
+ "clean_router": 0.00017077454611292685,
+ "slideshow": 0.00017077454611292685,
+ "locale": 0.00017077454611292685,
+ "mojom": 0.00017077454611292685,
+ "webkit_inspection_protocol": 0.00017077454611292685,
+ "mojo_apptest": 0.00017077454611292685,
+ "mutation_observer": 0.00017077454611292685,
+ "dictionary": 0.00017077454611292685,
+ "event_dispatcher": 0.00017077454611292685,
+ "emscripten": 0.00017077454611292685,
+ "patch_mirrors": 0.00017077454611292685,
+ "annotate": 0.00017077454611292685,
+ "domelement": 0.00017077454611292685,
+ "w_service": 0,
+ "ds": 0,
+ "dye": 0,
+ "query_builder": 0,
+ "jsonml": 0,
+ "editor_build": 0,
+ "mutable_string": 0,
+ "json_web_token": 0,
+ "sky_tools": 0,
+ "dart_events": 0,
+ "bass": 0,
+ "a_star": 0,
+ "dom_stub": 0,
+ "asset_pack": 0,
+ "requestful": 0,
+ "my_complex": 0,
+ "material_design_icons": 0,
+ "fo_components": 0,
+ "bwu_sparkline": 0,
+ "sky_engine": 0,
+ "streamable": 0,
+ "dispatch": 0,
+ "js_util": 0,
+ "dapter": 0,
+ "bbs": 0,
+ "preprocessor": 0,
+ "highcharts_options": 0,
+ "cupid": 0,
+ "jaguar_facebook_client": 0,
+ "multi_reg_exp": 0,
+ "newton": 0,
+ "sync_socket": 0,
+ "hub": 0,
+ "tentacle_response_formatter": 0
```

Comment:
We're not running on dart2js but rather on the VM, which has the proper number semantics.

```
 % cat test.dart
main() => print(1 as double);

 % dart test.dart
Unhandled exception:
type 'int' is not a subtype of type 'double' in type cast where
  int is from dart:core
  double is from dart:core

#0      Object._as (dart:core-patch/object_patch.dart:73)
#1      main (file:///usr/local/google/home/kustermann/repositories/vipunen-sdk/sdk/test.dart:1:19)
#2      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
```